sass/includes/_event-meta.scss

```diff
@@ -7,7 +7,7 @@
     margin-bottom: 2rem;
 
     @include media.on-larger-than-mobile {
-        padding-left: 3rem;
+        padding-left: 2rem;
```

Comment:
I know @ahosgood that you queried this one, but I've just double checked the designs and it is 3 rem rather than 2 rem here.

Reply:
I raised it because the guttering in other pages and all throughout the grid system is `2rem` so it didn't make sense to do something different. I'm trying to standardise this so we don't have a whole load of spacing that varies by tiny amounts.

What might be even better is if we use the gutter definition itself:

```scss
@use "@nationalarchives/frontend/nationalarchives/variables/grid" as gridVars;

.rule {
  padding-left: gridVars.$gutter-width
}
```

Reply:
That's fair enough thanks @ahosgood - I've updated this instance but sounds like it'll need a more in depth review to replace existing rems later.